src/components/new-request/search.vue

```diff
@@ -563,6 +578,20 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin, Sear
     return false
   }
 
+  /** Retrieve text for COLIN button based on selected action/flow */
+  get colinButtonText (): string {
+    var colinText = ''
```

Comment:
If this getter is used when this is not a conversion/restoration/change-name then it will return an incomplete string. I recommend initializing this string to "use this NR" as a safe fallback value.